build/install-dependencies.sh

```diff
@@ -10,6 +10,20 @@ if ! which golangci-lint > /dev/null; then
    curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.23.6
 fi
 
+# Install kubebuilder for unit test
```

Comment:
Is kubebuilder really needed here because I don't see any other reference to the tool except in this scripts.
If needed, Please wrap this in a "if" to not install it again if already installed

Reply:
`kubebuilder` is needed to set up the `webhook_suite_test.go` test env.

I will wrap it up to a if block